day7/sheldon/index.css

```diff
@@ -0,0 +1,410 @@
+    background-color: white;
+  }
+
+  .list__head {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 50px 32px 32px 56px;
+  }
+
+  .head__container {
+    max-width: 90%;
+    width: 100%;
+    margin: auto;
+    border-bottom: 2px solid rgba(0, 0, 0, 0.1);
+  }
+
+  button {
+    display: flex;
+    color: #ea3f58;
+    align-items: center;
+    background-color: #fff;
+    border: 1px solid #f1f2f4;
+    border-radius: 30px;
+    outline: none;
+    box-sizing: border-box;
+  }
+
+  .logOut {
+    margin: 16px 16px 17px 18px;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 19px;
+    box-sizing: border-box;
+  }
+
+  button svg {
+    padding-left: 18px;
+  }
+
+  .school {
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 17px;
+    color: rgba(131, 131, 131, 0.5);
+    margin-bottom: 8px;
+  }
+
+  .list__date {
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 28px;
+    color: black;
+  }
+
+  .green {
+    color: aquamarine;
+  }
+
+  .oncomingclass__class {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: black;
+  }
+
+  .oncomingclass__room {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 20px;
+    color: rgba(170, 170, 170, 0.5);
+  }
+
+  .oncomingclass {
+    padding: 50px 32px 32px 0px;
+    display: flex;
+    align-items: center;
+    max-width: 500px;
+    border-bottom: 2px solid rgba(0, 0, 0, 0.1);
+    margin-left: 57px;
+  }
+
+  .order {
+    display: flex;
+    align-items: center;
+    height: 58px;
+    width: 58px;
+    justify-content: center;
+    color: #ea3f58;
+    background-color: #fff;
+    border-radius: 50%;
+    margin-right: 25px;
+    border: 2px solid #b6bfc9;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  .detail {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: linear-gradient(246.21deg, #183d8e -43.38%, #762da7 102.31%);
+    width: 33vw;
+    height: 100vh;
+    position: fixed;
+  }
+
+  .list {
+    background-color: white;
+    width: 67%;
+    position: relative;
+    left: 33vw;
+    background-color: white;
+    height: auto;
+  }
+
+  .logo {
+    position: absolute;
+    padding: 0px;
+    top: 32px;
+    left: 24px;
+  }
+
+  .detail__ring {
+    margin-right: 0px;
+  }
+
+  .detail__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .view__block__title {
+    text-align: center;
+  }
+
+  .prograssbar {
+    position: absolute;
+    bottom: 40px;
+    left: 50%;
+    transform: translateX(-50%);
+    background-color: white;
+    color: black;
+    width: 80%;
+  }
+
+  .prograssbar__bar {
+    background-color: #f1f2f4;
+  }
+
+  .prograss {
+    background-color: #ea3f58;
+  }
+
+  .class {
+    margin-top: 32px;
+    font-size: 24px;
+    line-height: 24px;
+    text-align: center;
+    color: #fff;
+  }
+
+  .room {
```

Comment:
this is the second time i'm seeing class and room here, whats going on?

Reply:
on different screen boss